components/List/list.css

```diff
@@ -0,0 +1,9 @@
+.list-group{
+    list-style: none;
+    padding: 1rem;
+}
+
+.list-group li{
+    padding: 0.5rem 1rem;
+}
```

Comment:
One suggestion don't target any element direct or give style to that try to use as a class name.

Reply:
fixed it!